### PR TITLE
Boyer Moore’s Voting Algorithm in C++

### PR DESCRIPTION
**Pull request by ghost**
-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

This is my first pull request as my contribution to WoC.
Boyer Moore’s Voting Algorithm in C++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Its the solution to Boyer Moore’s Voting Algorithm in C++

### CODE REVIEW

**Comment by itskrypton** [comment]
Issue Number : #1439 
And Issue Number : #1443 
@iamrajiv Please let me know if there are any issues, Thank you

**Comment by ghost** [comment]
@iamrajiv The logo was also in this branch, sorry about that, ill take care next time, could you check that too?

**Review by iamrajiv — CHANGES_REQUESTED** [review]
Rename the file to `Boyer_Moore_Majority_Vote.cpp`. Add sample input-output, space complexity and time complexity.

**Comment by ghost** [comment]
Okay sure

**Comment by iamrajiv** [comment]
@itskrypton I requested for SVG logo, not PNG. Also, `Neo` and `Algo` should be together something like this `NeoAlgo`.

**Comment by ghost** [comment]
@iamrajiv Kindly review it now.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
Update the README.

**Comment by ghost** [comment]
Done.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
Update `C-Plus-Plus/README.md` file by adding the program link to it.

**Comment by ghost** [comment]
Sorry didn't know what to do, did it now.

File: C-Plus-Plus/Algorithms/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Boyer-Moore's Voting Algorithm
```
Comment (iamrajiv) [inline C-Plus-Plus/Algorithms/README.md:1]: @itskrypton Remove this file.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/Algorithms/README.md (hunk 1 above)
Comment (itskrypton) [reply C-Plus-Plus/Algorithms/README.md:1]: Okayy

Comment (itskrypton) [reply C-Plus-Plus/Algorithms/README.md:1]: The entire README?

**Review by iamrajiv — CHANGES_REQUESTED** [review]
Your code is similar to this https://www.geeksforgeeks.org/majority-element/.

**Comment by ghost** [comment]
But I wrote this a while ago for a competition at my college

**Comment by itskrypton** [comment]
Do you want me to change the code? I could do that.

**Comment by ghost** [comment]
I will rewrite this and submit another pull request

**Review by iamrajiv — CHANGES_REQUESTED** [review]
Again your code is similar to this https://www.techiedelight.com/find-majority-element-in-an-array-boyer-moore-majority-vote-algorithm/.

**Pull request closed without merging**